src/index.js

```diff
@@ -3,10 +3,17 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import ErrorPage from './components/ModelViewer/ErrorPage';
+
+
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <BrowserRouter>
+  <Routes>
+  <Route Component={App} path="/" />
+  <Route Component={ErrorPage} path="/*" />
```

Comment:
The Error page is not being rendered in the preview on non-existing routes probably because you've written `/*` in the path instead of just `*`. 

Kindly correct it and check the preview that gets generated. After that, I'll be happy to merge this PR.

Reply:
I just found it might also be the `Netlify` problem which shows `Page Not Found` when we enter the URL manually, but it should work on internal app routes. If that's the case then let me know if you're interested in solving that in separate PR.
For this PR just make the small change mentioned above considering the common practices for the 404 page route.